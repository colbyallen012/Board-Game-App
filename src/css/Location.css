article{
  border: 1px solid gainsboro;
  border-radius: 15px;
  margin: 10px;
  height: 130px;
}