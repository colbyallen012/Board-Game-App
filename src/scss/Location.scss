.locationCard {
  display: flex;
  flex-flow: column wrap;
  margin: 10px;
  margin-left: 0;
  margin-right: 100px;
  height: 130px;
}

.locationLogo {
  display: inline;
  border: 3px solid #000000;
  height: 75px;
  width: 75px;
}

.like-btn {
  margin: 6px;
  justify-content: flex-start;
}

.locationInfo {
  margin: 0px;
  display: inline;
}

p {
  font-size: 16px;
  flex-flow: wrap;
  margin-left: 0;
  margin-right: 20px;
}

.location-wrapper {
  border: 2px solid gainsboro;
  border-radius: 15px;
}
