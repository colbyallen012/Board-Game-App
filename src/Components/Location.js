import React, { Component } from 'react';
import '../scss/Location.scss';


function Location(props) {
  return(
    <div>
      <article className="locationCard">
        <img src={props.logo} className="locationLogo" />
        <img src={props.unlike} className="like-btn"></img>
        <div className="locationInfo">
          <h3>{props.name}</h3>
          <p>{ props.address }</p>
        </div>
      </article>
    </div>
    )
}

export default Location;
