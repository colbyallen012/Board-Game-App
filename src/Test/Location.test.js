import React from 'react';
import ReactDOM from 'react-dom';
import Location from '../Components/Location';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });



describe('Location', () => {
  let wrapper;
  
  beforeEach(() =>{
    wrapper = shallow(
      <Location/>
    )
  })

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});