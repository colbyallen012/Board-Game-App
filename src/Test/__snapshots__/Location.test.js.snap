// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location should match the snapshot with all data passed in correctly 1`] = `
<div
  className="location-wrapper"
>
  <article
    className="locationCard"
  >
    <img
      className="locationLogo"
    />
    <div
      className="locationInfo"
    >
      <h2 />
      <p />
      <p>
        M-F: 
        -
      </p>
      <p>
        Sa & Su: 
        -
      </p>
      <a />
    </div>
    <LikeButton
      like="Like.svg"
      unlike="Unlike.svg"
    />
  </article>
</div>
`;
